packages/contracts/contracts/libs/v0.8.x/minter-libs/AuthLib.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
@lyaunzbe - heads-up that we moved this file up one directory in the SEA minter branch, which was merged to main. 🙏

Reply:
roger roger, ty for the heads up 🙏